usr/share/lib/img_proof/tests/SLES/test_sles_kernel_version.py

```diff
@@ -0,0 +1,10 @@
+import pytest
+
+def test_sles_kernel_version(host, get_release_value):
+    version = get_release_value('VERSION')
+    assert version
+    version = version.split('-SP')
+    config = host.run('sudo zcat /proc/config.gz')
+    assert 'CONFIG_SUSE_VERSION={}\n'.format(version[0]) in config.stdout
+    if len(version) > 1 :
```

Comment:
extra space before `:`.

Reply:
sorry I run flake8 on the wrong folder ...

I changed that and also some other flake8 hints.